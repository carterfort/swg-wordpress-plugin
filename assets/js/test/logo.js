import Logo from '../logo';
import React from 'react';
import renderer from 'react-test-renderer';

it( 'renders correctly', () => {
  const logo = renderer
    .create(<Logo />)
    .toJSON();
  expect( logo ).toMatchSnapshot();
} );
