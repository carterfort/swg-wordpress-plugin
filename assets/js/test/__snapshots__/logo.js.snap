// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  aria-hidden="true"
  className="googlesitekit-logo"
>
  <svg
    className="svg googlesitekit-logo__logo-g"
    height="34"
    width="32"
  >
    <use
      xlinkHref="undefinedsvg/svg.svg#logo-g"
    />
  </svg>
  <svg
    className="svg 
				 googlesitekit-logo__logo-sitekit
				"
    height="26"
    width="99"
  >
    <use
      xlinkHref="undefinedsvg/svg.svg#logo-sitekit"
    />
  </svg>
  <span
    className="screen-reader-text"
  >
    Site Kit by Google Logo
  </span>
</div>
`;
